Ports/nesalizer/patches/0001-Disable-backtracing.patch

```diff
@@ -0,0 +1,42 @@
+From c8f67b656703ea4c98413b87b07650ed358c7f8b Mon Sep 17 00:00:00 2001
+From: Dan MacDonald <allcoms@gmail.com>
+Date: Sun, 8 Dec 2019 17:25:35 +0000
+Subject: [PATCH 1/2] Disable backtracing
+
+---
+ src/common.cpp | 6 +++---
+ 1 file changed, 3 insertions(+), 3 deletions(-)
+
+diff --git a/src/common.cpp b/src/common.cpp
+index 33df846..104e36a 100644
+--- a/src/common.cpp
++++ b/src/common.cpp
+@@ -1,6 +1,6 @@
+ #include "common.h"
+ 
+-#include <execinfo.h>
++// #include <execinfo.h>
+ #include <signal.h>
+ 
+ //
+@@ -96,7 +96,7 @@ void errno_fail(int errno_val, char const *format, ...) {
+ static void fatal_signal_handler(int sig) {
+     // Use non-async-signal-safe functions. Likely to work in practice.
+ 
+-    static void *backtrace_buffer[100];
++    /*static void *backtrace_buffer[100];
+     static char addr2line_cmd_buf[100];
+ 
+     fprintf(stderr, "caught fatal signal '%s'. Backtrace:\n\n", strsignal(sig));
+@@ -124,7 +124,7 @@ static void fatal_signal_handler(int sig) {
+             abort();
+         }
+ 
+-    abort();
++    abort();*/
```

Comment:
er. shouldn't we end the comment before the abort to make sure this function is actually noreturn?

Reply:
Yes, that makes more sense.